lib/index-static.jsx

```diff
@@ -35,6 +36,7 @@ function generateWithPageHTML(url, options, pageHTML) {
         <meta charSet="utf-8"/>
         <meta name="viewport" content="width=device-width, initial-scale=1"/>
         <meta property="og:description" content="We want more people to see themselves as citizens of the web. The Mozilla Learning Network offers programs and a global community dedicated to helping people learn the most important skills of our age: the ability to read, write and participate in the digital world."/>
+        <meta property="og:image" content={config.ORIGIN + "/img/pages/home/hero-unit.jpg"}/>
```

Comment:
This is really annoying, facebook doesn't rebase the URL here based on the current origin, it just expects absolute URLs, so we have to absolute-ify it ourselves. Unfortunately our link-checker wasn't built for this so I had to modify it to deal with this crap.